jmetal-core/src/main/java/org/uma/jmetal/solution/impl/DefaultDoubleSolution.java

```diff
@@ -77,4 +74,11 @@ public DefaultDoubleSolution copy() {
   public String getVariableValueString(int index) {
     return getVariableValue(index).toString() ;
   }
+  
+  private void initializeDoubleVariables() {
+    for (int i = 0 ; i < problem.getNumberOfVariables(); i++) {
+      Double value = randomGenerator.nextDouble(getLowerBound(i), getUpperBound(i)) ;
```

Comment:
Yes, that's was what I thought. Also, the ```DefaultIntegerDoubleSolution``` is implemented in the same fashion. So, I don't think this will be a problem.